File: app/src/main/java/com/popalay/cardme/data/repositories/DeviceRepository.java
package com.popalay.cardme.data.repositories;

import com.github.tamir7.contacts.Contact;
import com.github.tamir7.contacts.Contacts;

import java.util.List;

public class DeviceRepository implements IDeviceRepository {

    @Override public List<Contact> getContacts(){
        return Contacts.getQuery().find();
    }
}

File: app/src/main/kotlin/com/popalay/cardme/data/repositories/device/DeviceRepository.kt
package com.popalay.cardme.data.repositories.device

import com.github.tamir7.contacts.Contact
import com.github.tamir7.contacts.Contacts

import javax.inject.Inject
import javax.inject.Singleton

@Singleton
class DeviceRepository @Inject constructor() {

    fun getContacts(): List<Contact> = Contacts.getQuery().find()
}
